### DISCUSSION TITLE
Html markup shown in preview

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#49598] [Mobile] [answered]
GitHub Mobile beta 
1.  visit https://github.com/community/community/discussions/9566
2. See a comment preview with nbsp
    ![Screenshot_20230309-023540.png](https://user-images.githubusercontent.com/2119212/223952700-c16615a1-eb61-49d6-abf8-1408407bbabd.png)
3. Tap the thread
    ![Screenshot_20230309-023553.png](https://user-images.githubusercontent.com/2119212/223952660-f6a13773-01d4-47f3-970e-2d266e781dd3.png)

Expect the original preview to be similar to the later view

**Comment by aconsuegra** [comment]
Thanks for the report @jsoref! We created an issue for our engineers, I hope we can ship a fix soon.

**Comment by stevepopovich** [comment, marked as answer]
Hi @jsoref, this should be fixed in the next release. Thanks!
